simplyblock_core/storage_node_ops.py

```diff
@@ -817,8 +817,8 @@ def add_node(cluster_id, node_addr, iface_name, data_nics_list,
         if not cloud_instance:
             # Create a static cloud instance from node info
             cloud_instance = {"id": node_info['system_id'], "type": "None", "cloud": "None",
-                              "ip": node_info['network_interface'][iface_name]["ip"],
-                              "public_ip": node_info['network_interface'][iface_name]["ip"]}
+                              "ip": utils.get_mgmt_ip(node_info, iface_name),
```

Comment:
now that variable `iface_name` takes multiple values. should we also update the variable to represent that its a list?

Reply:
You mean the naming should be changed to **iface_names**